go.mod

```diff
@@ -12,15 +12,14 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/fsouza/go-dockerclient v1.10.0
 	github.com/go-kit/kit v0.10.0
-	github.com/golang/protobuf v1.5.4
 	github.com/gorilla/handlers v1.5.1
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
 	github.com/hyperledger-labs/SmartBFT v0.0.0-20240616160543-3f61a410b8c1
-	github.com/hyperledger/fabric-chaincode-go v0.0.0-20220713164125-8f0791c989d7
+	github.com/hyperledger/fabric-chaincode-go/v2 v2.0.0-20240802023949-a356b32676fd
```

Comment:
A proper v2.0.0 release is now available - https://github.com/hyperledger/fabric-chaincode-go/releases/tag/v2.0.0